front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/SurveyQuestionResultWidget/Question/utils.ts

```diff
@@ -41,3 +51,27 @@ export const generateDateRange = (
 export const getPhaseIndex = (phases: IPhaseData[], phaseId: string) => {
   return phases.findIndex((phase) => phase.id === phaseId);
 };
+
+const EMPTY_COLOR = colors.coolGrey300;
+
+export const getColorScheme = (length: number) => {
+  if (length > 11) {
```

Comment:
I still need to figure out a way that makes it impossible to select custom fields / survey questions with more than 10 options (11 if you include the 'nil' option)